README.md

```diff
@@ -30,7 +30,7 @@ Violation of the DMCA or Copyright law is the responsibility of the submitter.
 ## Primary Data Source
 
 We attempt to derive from machine sources and produce machine readable files (YAML) in this repo under `_data`.  For
-information about creating and extending data format see [Data Format Guidance](docs/Data_Formats).
+information about creating and extending data format see [Data Format Guidance](docs/Data_Formats.md).
```

Comment:
So md would make it work in GitHub, but I think breaks the link on the web.  Did you run Jekyll to verify this works?